secrets/dataplane-observability-config.yaml.sample

```diff
@@ -0,0 +1,40 @@
+---
+# [required]
+remote_write_configuration:
+  # [required] V2 URL endpoint of the Prometheus instance to be used by the data planes
```

Comment:
This `V2` is not needed. It is a remain from when we were prototyping the configuration